takoyaki/src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +14,22 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+  public static final class kIntake {
+    public static final int FALCON_MOTOR_ID = -1;
+    public static final int SOLENOID_FRONT = 1;
+    public static final int SOLENOID_BACK = 0;
+    public static final float INTAKE_SPEED = 0.9f;
+
+    public static final int RETRACT_INTAKE = XboxController.Button.kY.value;
+    public static final int ACUATE_INTAK = XboxController.Button.kA.value;
```

Comment:
nit: spelling: actuate intake instead of acuate intak
```suggestion
    public static final int ACTUATE_INTAKE = XboxController.Button.kA.value;
```